st2common/st2common/operators.py

```diff
@@ -161,6 +169,7 @@ def nexists(value, criteria_pattern):
     return value is None
 
 # operator match strings
+MATCH_GLOB = 'matchglob'
```

Comment:
I donno why someone would use match_glob in rules context unless they want glob matching on filenames which I think can be done with match_regex.